research/blanks/model.py

```diff
@@ -124,13 +125,26 @@ def get_ff_layer(args):
         raise NotImplementedError(f"ff_mode {args.ff_mode} not implemented")
 
 
-def get_attention_layer(args):
+def get_custom_attention_layer(args):
+    attention_layer_fun = lambda: BlankAttention(
+        dmodel=args.dmodel,
+        heads=args.n_att_heads,
+        dhead=args.dhead,
+        flash=False,
+        init_type=args.init_type,
+        init_scale=args.init_scale,
+    )
+
+    return attention_layer_fun
+
+
+def get_normal_attention_layer(args):
     attention_layer_fun = lambda: llm.Attention(
         dmodel=args.dmodel,
         heads=args.n_att_heads,
-        causal=True,
         dhead=args.dhead,
         flash=True,
+        causal=True,
         init_type=args.init_type,
         init_scale=args.init_scale,
     )
```

Comment:
Let's unify this and add an `if` for either choosing BlankAttention or llm.Attention (see how they do that in conditional.

Reply:
Done